Compute Shaders/main.cpp

```diff
@@ -88,7 +79,7 @@ int main()
 
 	glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, OPENGL_MAJOR_VERSION);
 	glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, OPENGL_MINOR_VERSION);
-	glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE);
+	glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_COMPAT_PROFILE);
```

Comment:
I have to request a compatibility profile here because it's drawing with `GL_QUADS` and there is no VAO bound.
Both of which are not allowed in core but could easily be adjusted to be used in a core context.